src/app/teamSheets/_TeamSheetsBlankState/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link'
+
+const TeamSheetListItem = () => {
+
+
+  return (
+    <div>
+      <div>
+        You haven&apos;t created any team sheets yet.
+      </div>
+
+      <Link href="/">
```

Comment:
probs better to send the user to `/sport`, also we should remove the Football record from the db since we have no setup for it